Realm/Realm/Realm.csproj

```diff
@@ -61,12 +61,6 @@
     </None>
     <!-- macOS -->
     <Content Include="wrappers-props\xamarinmac\wrappers.props" Pack="true" PackagePath="build\xamarinmac\Realm.props" />
-    <None Include="..\..\wrappers\build\Darwin\$(Configuration)\librealm-wrappers.dylib">
```

Comment:
Is this removed because of Catalyst?

Reply:
No, this seems like something Visual Studio removed accidentally. Thanks for spotting it!